tests/pipelines/flux/test_pipeline_flux.py

```diff
@@ -136,12 +138,13 @@ def test_flux_prompt_embeds(self):
         inputs = self.get_dummy_inputs(torch_device)
         prompt = inputs.pop("prompt")
 
-        (prompt_embeds, pooled_prompt_embeds, text_ids) = pipe.encode_prompt(
+        prompt_outputs = pipe.encode_prompt(
```

Comment:
We had copied `encode_prompt` from the cfg pipeline which returns extra values, it's reverted now because we use the original `encode_prompt` twice instead.